packages/openapi-ts/test/__snapshots__/3.1.x/clients/@hey-api/client-nuxt/bundle/client/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import { AsyncDataOptions, useAsyncData, useFetch, useLazyAsyncData, useLazyFetch, UseFetchOptions } from 'nuxt/app';
+import { UseFetchOptions, AsyncDataOptions, useAsyncData, useFetch, useLazyAsyncData, useLazyFetch } from 'nuxt/app';
```

Comment:
I had this issue before and forgot how I fixed it that time... very annoying as it fails tests!